apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/site-logo.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { addFilter } from '@wordpress/hooks';
+
+/**
```

Comment:
We don't need this comment block if there are no internal dependencies afaik.

Reply:
The WP dependencies block doesn't clear the ESLint import rule.
In other cases (back in the Gutenlypso days) I've solved it by disabling the rule:
```
/* eslint-disable wpcalypso/import-docblock */
```

Reply:
the pre-commit lint rule is a bit broken for this, as Jacopo notes it is not happy with just the WP deps one, but have fixed it with his eslint-disable instead for now.